packages/arb-token-bridge-ui/src/hooks/useArbQueryParams.tsx

```diff
@@ -174,7 +174,8 @@ export function ArbQueryParamProvider({
         searchStringToObject: queryString.parse,
         objectToSearchString: queryString.stringify,
         updateType: 'replaceIn', // replace just a single parameter when updating query-state, leaving the rest as is
-        removeDefaultsFromUrl: true
+        removeDefaultsFromUrl: true,
+        enableBatching: true
```

Comment:
We shouldn't use enableBatching in production as it is marked `experimental` in `use-query-params` API. 
Reference: https://www.npmjs.com/package/use-query-params#:~:text=false-,experimental,-%2D%20turns%20on%20batching

Reply:
agreed with @chrstph-dvx